composer.json

```diff
@@ -25,7 +25,8 @@
     "require": {
         "php": ">=5.6",
         "ext-iconv" : "*",
```

Comment:
Looks like it, a quick global search for 'iconv' shows up nothing, which I believe means we can get rid of the requirement.

I'll remove this now.